### PR TITLE
Add support for Fakro Roof Windows Roller Shutter ARZ Z-Wave

### PR DESCRIPTION
**Pull request by pepeEL**

### CODE REVIEW

File: config/fakro/arz-z-wave.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Product xmlns='http://code.google.com/p/open-zwave/'>
+  <!-- Configuration -->
+  <CommandClass id="112">
```
Comment (Fishwaldo) [inline config/fakro/arz-z-wave.xml]: If there is no configuration items, then this section can be dropped.

File: config/fakro/arz-z-wave.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  </CommandClass>
+
+  <!-- Association Groups -->
+  <CommandClass id="133">
```
Comment (Fishwaldo) [inline config/fakro/arz-z-wave.xml]: You should put in Association Information.

File: config/fakro/arz-z-wave.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    <Associations>
+    </Associations>
+  </CommandClass>
+
+</Product>
```
Comment (Fishwaldo) [inline config/fakro/arz-z-wave.xml]: If the file really ends up "empty" then we can drop the file entirely.

File: config/fakro/arz-z-wave.xml (hunk 2 above)
Comment (pepeEL) [inline config/fakro/arz-z-wave.xml]: But from where i can get Association Group ? Becuase now when i use it in domoticz it has no association group.
ass

File: config/fakro/arz-z-wave.xml (hunk 1 above)
Comment (pepeEL) [inline config/fakro/arz-z-wave.xml]: And from where i can get option to configuration ? In manual from roller i have no option to config.

**Comment by nechry** [comment]
Hi the ARZ Roof Window Roller Shutter don't have association groups or configuration parameters. we tested this device. see:
 https://github.com/jeedom/plugin-openzwave/blob/master/ressources/openzwave/config/manufacturer_specific.xml#L265

the file config/fakro/arz-z-wave.xml is not require.

**Comment by pepeEL** [comment]
Ok thanks i correct it in new PR #760

**Pull request closed without merging**